projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -4703,6 +4703,187 @@ public string FatherIndustry
             set => SetIndustry("FTH", value);
         }
 
+
+        private Observation SetHeight(int? value)
```

Comment:
There's no other field that needs to use this SetHeight function correct? This code can just be moved to the MotherHeight setter.

Reply:
It also gets used by `SetHeightEditFlag` to create a null hieght if none is present